src/utils.rs

```diff
@@ -62,3 +63,14 @@ pub(crate) fn number_of_derived_accounts(path: &Path) -> usize {
         0
     }
 }
+
+pub(crate) fn derive_account_with_index(path: &Path, account_index: usize) -> Result<SecretKey> {
+    let password = rpassword::prompt_password(
+        "Please enter your password to decrypt initialized wallet's phrases: ",
+    )?;
+    let phrase_recovered = eth_keystore::decrypt_key(path.join(".wallet"), password)?;
+    let phrase = String::from_utf8(phrase_recovered)?;
+    let derive_path = format!("m/44'/1179993420'/{}'/0/0", account_index);
```

Comment:
Perhaps we should be using an upstream `const` and/or function for constructing this path? E.g. at least a `wallet_account_derive_path` function or similar. It's probably fine for now, but I've opened an issue to follow-up: https://github.com/FuelLabs/forc-wallet/issues/28.